src/script/lua_api/l_item.cpp

```diff
@@ -525,6 +525,29 @@ int ModApiItemMod::l_register_item_raw(lua_State *L)
 	return 0; /* number of results */
 }
 
+// unregister_item(name)
+int ModApiItemMod::l_unregister_item_raw(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	std::string name = luaL_checkstring(L, 1);
+	
+	// Get the writable item definition manager from the server
+	IWritableItemDefManager *idef =
+			getServer(L)->getWritableItemDefManager();
+	
+	// Unregister the node
+	if(idef->get(name).type == ITEM_NODE){
```

Comment:
That's generally not a bad idea but these parts do not follow the code style guidelines (yet). Each commit only changes a specific part of the code, thus it will take a while to unify the code style in this project.